src/Documentation/doc/advanced/corporation/faq.md

```diff
@@ -193,8 +193,13 @@ As soon as possible, it greatly increases your profit because it can find the op
 
 #### What is the difference between Market-TA1 and Market-TA2?
 
-Market-TA1: set a price that ensures that you can sell all produced goods in storage.  
-Market-TA2: set the highest possible price that ensures that you can sell all produced goods in storage.
+If you set the price of the material/product too much higher than the market price, you will get a penalty modifier that makes the material/product not be able to be sold as much as it should be.
+
+Market-TA1: Set a price higher than the market price while ensuring that you don't get the penalty modifier.
+
+Market-TA2: Set the highest possible price while ensuring that you don't get the penalty modifier.
```

Comment:
Isn't this not quite correct? I thought TA1 stops at the point where the penalty modifier kicks in, whereas TA2 increases price to match sales to the requested sellAmt (and this price is usually well past the onset of the penalty modifier).